src/scss/components/css3/_background-gradient.scss

```diff
@@ -0,0 +1,34 @@
+@charset "UTF-8";
+
+/// Background gradient helper
+///
+/// @access public
+///
+/// @param {Color} $start-color - Start color
+/// @param {Color} $end-color - End color
+/// @param {String} $orientation - Type of gradient, either `vertical`, `horizontal` or `radial`
+///
+/// @example scss - Usage
+///   .gradient {
+///     @include background-gradient(red, black, 'vertical');
+///   }
+///
+/// @example css - Result
+///   .gradient {
+///     background: -webkit-linear-gradient(top, red, black);
+///     background: linear-gradient(to bottom, red, black);
+///   }
+@mixin background-gradient($start-color, $end-color, $orientation) {
+    background: $start-color;
+
+    @if $orientation == 'vertical' {
+      background: -webkit-linear-gradient(top, $start-color, $end-color);
+      background: linear-gradient(to bottom, $start-color, $end-color);
+    } @else if $orientation == 'horizontal' {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Prefer double-quoted strings